spec/models/mechanic_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe Mechanic, type: :model do
+  describe "relationships" do
+    it { should have_many(:rides) }
```

Comment:
Remember to add the association tests for all 3 models involved in the many-to-many relationship! And it's best practice to have a test for each association, so here you'd want both:
`it { should have_many :ride_mechanics }`
`it { should have_many(:mechanics).through(:ride_mechanics)`